org.mixedrealitytoolkit.spatialmanipulation/BoundsControl/BoundsHandleInteractable.cs

```diff
@@ -102,6 +102,23 @@ protected override void Awake()
             HideOnStartup();
         }
 
+        /// <summary>
+        /// A Unity event function that is called every frame, if this object is enabled.
+        /// </summary>
+        protected virtual void Update()
+        {
+            // Do our IsOccluded "setter" in Update so we don't do this multiple times a frame.
+            if (IsOccluded != wasOccludedLastFrame)
```

Comment:
The SqueezeBoxVisuals update BoundsHandleInteractable::IsOccluded in Update(). To ensure that BoundsHandleInteractable has a valid IsOccluded value for the frame, BoundsHandleInteractable has to use IsOccluded  in LateUpdate() and not Update()....
 
So we shouldn't move this code.

Reply:
I think if you remove the call to SqueezableBoxVisuals::Update() in  SqueezableBoxVisuals::Awake() .... this code can be in LateUpdate().   That is, if SqueezableBoxVisuals::Update() is not called, then IsOccluded is not updated. Which means `if (IsOccluded != wasOccludedLastFrame)` won't evaluate to `true`.

Reply:
Opps... found another issue.  Since `SqueezeBoxVisuals::OnEnable` and  `BoundsHandleInteractable::Awake` calls `HideOnStartup()`, and `HideOnStartup()` sets `wasOccludedLastFrame` to `true`. `LateUpdate()` incorrectly uses the `IsOccluded` value too soon.  

We re-think `HideOnStartup` that was added with this PR....https://github.com/microsoft/MixedRealityToolkit-Unity/pull/11698

If we need to keep HideOnStartup, then should set IsOccluded to true in HideOnStartup...that should fix stuff